Kernel/init.cpp

```diff
@@ -187,6 +190,12 @@ extern "C" [[noreturn]] UNMAP_AFTER_INIT void init(BootInfo const& boot_info)
     CommandLine::initialize();
     Memory::MemoryManager::initialize(0);
 
+    if (!multiboot_framebuffer_addr.is_null()) {
+        // If the bootloader provided a framebuffer then set up an initial console so we can see the output on the screen!
+        boot_framebuffer_console = try_make_ref_counted<Graphics::BootFramebufferConsole>(multiboot_framebuffer_addr, multiboot_framebuffer_width, multiboot_framebuffer_height, multiboot_framebuffer_pitch).leak_ref();
```

Comment:
Can we make this to actually put text in VGA text mode too?

Reply:
I suppose we could, I'll leave that for a future improvement when I have some more time.